modules/storages/app/components/storages/admin/storage_list_component.sass

```diff
@@ -0,0 +1,27 @@
+.op-storage-list
+  display: grid
+  grid-template-columns: fit-content(180px) 1fr 180px 100px 240px
+  grid-template-areas: "name label user provider time" "host host user provider time"
```

Comment:
We have to do some fine-tuning at the grid I think. On smaller screens the timestamp overflows quite easily:
<img width="615" alt="Bildschirmfoto 2023-10-04 um 09 17 50" src="https://github.com/opf/openproject/assets/7457313/bd884ea1-d27f-418b-b960-efc8ac5a9b99">

Further, the label should probably not be the element that has `1fr` because it is not even visible in most cases. This leads to the name being cut off weirdly.
<img width="988" alt="Bildschirmfoto 2023-10-04 um 09 19 08" src="https://github.com/opf/openproject/assets/7457313/115f3079-d99b-43fe-af63-49b9752fe885">